griptape/schemas/base_schema.py

```diff
@@ -1,5 +1,63 @@
+from __future__ import annotations
+import builtins
+import attrs
 from marshmallow import Schema, fields
 
 
 class BaseSchema(Schema):
     schema_namespace = fields.Str(allow_none=True)
+    DATACLASS_TYPE_MAPPING = {**Schema.TYPE_MAPPING, list: fields.List}
+
+    @classmethod
+    def from_attrscls(cls, attrscls):
+        """Generate a Schema from an attrs class."""
+        from marshmallow import post_load
+        from importlib import import_module
+
+        class SubSchema(cls):
+            @post_load
+            def make_obj(self, data, **kwargs):
+                package_name = ".".join(attrscls.__module__.split(".")[:2])
```

Comment:
Need advice here. `attrscls` will be something like `<class 'griptape.drivers.prompt.base_prompt_driver.BasePromptDriver'>` and we need to get the packages its subclasses will exist in. I.e. `griptape.drivers`.

Is there a better way to do extract this info from a class? Its top level package where it is exported?